app/experimenter/experiments/constants/nimbus.py

```diff
@@ -195,118 +194,19 @@ class NimbusTargetingConfig:
 )
 
 
-class Status(models.TextChoices):
```

Comment:
> While working on a recent bug I discovered that one of our lifecycle states was malformed

Which one was that? This move obscures the change

Reply:
Oh sorry I didn't link there's no bugfix here, just moving the constants.  Bugfix was here https://github.com/mozilla/experimenter/issues/5351